### ISSUE TITLE
Galasactl tool to be able to retrieve results of a test run from the ecosystem

### ISSUE THREAD

**Issue by techcobweb** [galasa-dev/projectmanagement#1262]
## Story
As a galasa user running tests either locally or via automation, I want to use the command-line to get the results of a passed/failed test run, and see/explore the artifacts on my local system.

## Background
- If you kick off a test, you can't get the results back from an ecosystem without using eclipse currently.
- Eclipse gets the results by going directly to couchdb currently.
- We would need to add to the REST interfaces on the API server to allow GETs of test run result artifacts.

## Questions
- What would the syntax be ?
- Can we get the results summary ? or different levels of detail from a 1-liner result status to all of the artifacts ?
- Would it dump a zip somewhere ? or expanded files ? Can you control where it gets put ?
- How would we test it ?
- Do we need another tool to render the descriptions of the 3270 streams into images to be viewed easily ?
  - Should that be done server-side prior to the artifacts being downloaded ?

## Tasks
- [ ] Spike: Answer all the questions above so we have more detail before we start anything.
- [ ] Expand the api server to get results/artifacts via a REST API
- [ ] Enhance galasactl to get the artifacts using the REST API
- [ ] Test ???

**Comment by techcobweb** [comment]
Duplicate of story #803 so closing.
